### PR TITLE
Country-specific tooltips, a code nit pick

### PR DESCRIPTION
**Pull request by Jelena0921**
Adding country specific tooltips.

### CODE REVIEW

File: src/TrendsOverview.svelte [hunk 1]
```diff
@@ -48,6 +48,17 @@
   let regionalTrends: Map<string, RegionalTrends>;
   let selectedMapTrendId: string = "vaccination";
 
+  let vaccineTooltip = `
```
Comment (mhkshum) [inline src/TrendsOverview.svelte:51]: nit: add type ': string'

File: src/TrendsOverview.svelte [hunk 2]
```diff
@@ -48,6 +48,17 @@
...
+    Search interest in the eligibility, availability, and accessibility of
+    COVID-19 vaccines. `;
+  let intentTooltip = `
```
Comment (mhkshum) [inline src/TrendsOverview.svelte:54]: same nit

**Review by mhkshum — COMMENTED** [review]
Just small little nit. Otherwise, LGTM!

File: src/TrendsOverview.svelte (hunk 1 above)
Comment (Jelena0921) [reply src/TrendsOverview.svelte]: Done, thank you for spotting!

**Review by mimisun — APPROVED** [review]

**Pull request merged**